include/exadg/incompressible_navier_stokes/postprocessor/line_plot_calculation.h

```diff
@@ -67,6 +68,9 @@ class LinePlotCalculator
   SmartPointer<Mapping<dim> const>    mapping;
 
   LinePlotDataInstantaneous<dim> data;
+
+  std::vector<Point<dim>>                                     evaluation_points;
+  std::shared_ptr<Utilities::MPI::RemotePointEvaluation<dim>> evaluation_cache;
```

Comment:
I think we need a comment here. Both attributes `Remote` and `cache` are not really intuitive in my opinion. What exactly is the meaning?

Reply:
I agree on the evaluation concept, whereas I think the term *cache* should be self-explanatory in terms of the underlying software concept. Citing from [Wikipedia](https://en.wikipedia.org/wiki/Cache_(computing)), "In computing, a cache is a hardware or **software component** that stores data so that future requests for that data can be served faster; the **data stored in a cache might be the result of an earlier computation** or a copy of data stored elsewhere" [emphasis mine].

Reply:
ok, it's only a minor comment.